Assets/Mirror/Runtime/NetworkConnectionToClient.cs

```diff
@@ -16,6 +16,17 @@ public NetworkConnectionToClient(int networkConnectionId) : base(networkConnecti
         // the client. they would be detected as a message. send messages instead.
         readonly List<int> singleConnectionId = new List<int> { -1 };
 
+        /// <summary>
+        /// Number of seconds of no messages from client after which server will auto-disconnect
+        /// </summary>
+        public float serverIdleTimeout = 30f;
+
+        internal void CheckForActivity()
+        {
+            if (Time.time - serverIdleTimeout > lastMessageTime)
```

Comment:
```suggestion
            if (Time.time >= lastMessageTime + serverIdleTimeout)
```
addition is a bit easier to understand

Reply:
Uses same subtraction as NetworkTransform.